services/visual-studio-app-center/visual-studio-app-center-releases-size.service.js

```diff
@@ -47,14 +47,8 @@ export default class VisualStudioAppCenterReleasesSize extends BaseVisualStudioA
     color: 'blue',
   }
 
-  static render({ size }) {
-    return {
-      message: prettyBytes(size),
-    }
-  }
-
   async handle({ owner, app, token }) {
     const { size } = await this.fetch({ owner, app, token, schema })
-    return this.constructor.render({ size })
+    return renderSizeBadge(size, 'metric')
```

Comment:
I have no idea how this site presents sizes. Lets stick with metric for the moment on the basis we have been doing that for some time and nobody has complained yet. We can change it if we need to.